SPECS/rdma-core/rdma-core.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        RDMA core userspace libraries and daemons
 Name:           rdma-core
-Version:        39.0
-Release:        2%{?dist}
+Version:        49.0
```

Comment:
It seems we have missed to update `cgmanifest` entry :)

Reply:
Yes, will update it shortly. Thanks.